.github/scripts/test-info.mjs

```diff
@@ -232,6 +232,25 @@ const components = {
             'conformance-pubsub.aws.snssqs.terraform-destroy.sh',
         sourcePkg: 'pubsub/aws/snssqs',
     },
+    'pubsub.gcp.pubsub': {
+        certification: true,
+        requireTerraform: true,
+        requireGCPCredentials: true,
+        certificationSetup: 'certification-pubsub.gcp.pubsub-setup.sh',
+        certificationDestroy: 'certification-pubsub.gcp.pubsub-destroy.sh',
+    },
+    'pubsub.gcp.pubsub.docker': {
```

Comment:
Could we please comment out the docker case? Just like the Azure components we do not run a dockerized emulator, and it would be best to to minimize the number of task runners every PR executes.

Reply:
I made this update.